docs/cluster-deployment/stripe.rst

```diff
@@ -135,19 +136,42 @@ Provide billing information
 ===========================
 
 Finally, you will be taken to a new screen where you can fill out your billing
-information. Our payment processing is supported by `Stripe`_. At the bottom
-right you can find the cards accepted by Crate.io. When you have filled out the
-necessary information, click *Deploy* below it to deploy your cluster. Do not
-forget to accept financial authorization by ticking the box at the bottom.
+information. Our credit card payment processing is supported by `Stripe`_. If
+you have an existing AWS or Azure subscription, you can also pay using that.
+You can choose the payment method right after you configure your cluster and
+click **Next**.
+
+.. image:: ../_assets/img/payment-method.png
+   :alt: Payment method screen
+
+Credit card
+-----------
+
+You can find the cards accepted by Crate.io at the bottom right part of the screen.
+When you have filled out the necessary information, click *Deploy* below it to
+deploy your cluster. Do not forget to accept financial authorization by ticking
+the box at the bottom.
 
 .. image:: ../_assets/img/stripe-billing.png
    :alt: Billing information screen
 
+AWS/Azure
+---------
+
+The other payment option is to use AWS or Azure marketplace subscription.
+Simply choose the one you'd like to use and you will be brought to
```

Comment:
If you wanted to be more specific in your description perhaps you could say either
```suggestion
Simply choose the one you'd like to use and you will be taken to
```
or perhaps better still

```suggestion
Simply choose the one you'd like to use and you will be forwared to
```